style/swift/sample.swift

```diff
@@ -31,16 +31,19 @@ var maybe: Bool?
 
 // Use if-let syntax to unwrap optionals
 if let definitely = maybe {
-    println("This is \(definitely) here")
+    print("This is \(definitely) here")
 }
 
 // If the API you are using has implicit unwrapping you should still use if-let
 func someUnauditedAPI(thing: String!) {
     if let string = thing {
-        println(string)
+        print(string)
     }
 }
 
+// You can also use the `?` postfix operator to chain possible execution
+delegate?.doSomething()
```

Comment:
It specifically says that you should use `if let` even if you're dealing with an implicitly unwrapped optional. It's not talking about using `if let` generally.